src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -390,7 +393,7 @@ def forward(self, intermediate_output):
         )
 
 
-class YoloBase(SgModule):
+class YoloBase(SgModule, HasPredict):
```

Comment:
Same as above (I would go for protocol type checking)

Reply:
changed in https://github.com/Deci-AI/super-gradients/commit/d9e3019ce07736224b9e07687bc770e793a2c612